homeassistant/components/geo_json_events/geo_location.py

```diff
@@ -59,63 +60,72 @@ def setup_platform(
     )
     radius_in_km = config[CONF_RADIUS]
     # Initialize the entity manager.
-    feed = GeoJsonFeedEntityManager(
-        hass, add_entities, scan_interval, coordinates, url, radius_in_km
+    manager = GeoJsonFeedEntityManager(
+        hass, async_add_entities, scan_interval, coordinates, url, radius_in_km
     )
+    await manager.async_init()
 
-    def start_feed_manager(event):
+    @callback
```

Comment:
A coroutine function is never a callback.
```suggestion
```

Reply:
OK. Removed annotation.